platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/sources/CustomGeometrySource.java

```diff
@@ -103,15 +105,15 @@ public void setTileData(int zoomLevel, int x, int y, FeatureCollection data) {
    */
   @NonNull
   public List<Feature> querySourceFeatures(@Nullable Expression filter) {
-    checkThread();
```

Comment:
The annotation is now wrong.  Since the class is annotated as UiThread, everything is UiThread only be default.  Need to add AnyThread annotation.  Also may want to mention that it is useable from any thread in the JavaDoc.